docs/examples/micropayment.rst

```diff
@@ -180,26 +209,44 @@ The full contract
 Writing a Simple Payment Channel
 ================================
 
-Alice now builds a simple but complete implementation of a payment channel. Payment channels use cryptographic signatures to make repeated transfers of Ether securely, instantaneously, and without transaction fees.
+Alice now builds a simple but complete implementation of a payment
+channel. Payment channels use cryptographic signatures to make
+repeated transfers of Ether securely, instantaneously, and without transaction fees.
 
 What is a Payment Channel?
 --------------------------
 
-Payment channels allow participants to make repeated transfers of Ether without using transactions. This means that you can avoid the delays and fees associated with transactions. We are going to explore a simple unidirectional payment channel between two parties (Alice and Bob). It involves three steps:
+Payment channels allow participants to make repeated transfers of Ether
+without using transactions. This means that you can avoid the delays and
+fees associated with transactions. We are going to explore a simple
+unidirectional payment channel between two parties (Alice and Bob). It involves three steps:
 
     1. Alice funds a smart contract with Ether. This "opens" the payment channel.
     2. Alice signs messages that specify how much of that Ether is owed to the recipient. This step is repeated for each payment.
     3. Bob "closes" the payment channel, withdrawing their portion of the Ether and sending the remainder back to the sender.
 
 .. note::
-  Only steps 1 and 3 require Ethereum transactions, step 2 means that the sender transmits a cryptographically signed message to the recipient via off chain methods (e.g. email). This means only two transactions are required to support any number of transfers.
-
-Bob is guaranteed to receive their funds because the smart contract escrows the Ether and honours a valid signed message. The smart contract also enforces a timeout, so Alice is guaranteed to eventually recover their funds even if the recipient refuses to close the channel. It is up to the participants in a payment channel to decide how long to keep it open. For a short-lived transaction, such as paying an internet café for each minute of network access, or for a longer relationship, such as paying an employee an hourly wage, a payment could last for months or years.
+  Only steps 1 and 3 require Ethereum transactions, step 2 means that the sender
```

Comment:
Is the extra indentation in this and the next few lines intended?

Reply:
Should be the same indentation as the non-broken line was before.

Reply:
The second line onwards are left aligned in the original text but not in this PR, is that also intended?